Adding Rune Tap ability to Blood Deathknights

ReneM opened the pull request:
Please be very critical, since I'm not currently able to compile and test this.
This is coded purely on looking at the surrounding code, so it's very likely I'm missing something.

The purpose, is to add a new option for Death Knights, that enables to use of [Rune Tap](https://wotlkdb.com/?spell=48982).
It's currently enabled to true - I'm unsure if that's the best choice?

If the option is enabled, it should use Rune Tap when below 50% HP.

MrCeeJ commented:
I would make the hp% configurable in the settings (there are plenty of examples in the code) and lower the priority since it is a survivability setting, no point in doing DPS or gaining threat if you are in danger if dying

ReneM commented:
> I would make the hp% configurable in the settings (there are plenty of examples in the code) and lower the priority since it is a survivability setting, no point in doing DPS or gaining threat if you are in danger if dying

Great idea!
I've changed the option to be percentage of health instead
![image](https://user-images.githubusercontent.com/483707/186991053-cbc149e0-e9a4-4ebb-8b1b-ea248b94b80d.png)

I misunderstood the priority system, moved it much lower in priority now

I also, managed to get it compiled so I can test it :)

During tests, the bot uses it at the set HP level:
![image](https://user-images.githubusercontent.com/483707/186993528-f05f925d-e5ab-4b70-9f33-fdb87c25d3d3.png)

I've had it running for some 10 minutes, and its behaving as expected.

MrCeeJ commented:
Seems good, @Talamin fancy merging it?
